help/destinations/catalog/crm/zendesk.md

```diff
@@ -0,0 +1,158 @@
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Zendesk] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follow these steps:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
+        |Source Field|Target Field| Mandatory|
+        |---|---|---|
+        |`xdm: person.name.lastName`|`Attribute: last_name`| Yes, Note you can also use `name`. |
+        |`IdentityMap: Email`|`Identity: email`| Yes |
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/crm/zendesk/mappings.png)
+
+>[!IMPORTANT]
+>
+>You must have a mapping for *Last Name* or *Name* otherwise the [!DNL Zendesk] API does not respond with an error, and attribute value ignored.
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+### Schedule segment export and example {#schedule-segment-export-example}
+
+In the [[!UICONTROL Schedule segment export]](/help/destinations/ui/activate-segment-streaming-destinations.md#scheduling) step of the activation workflow, you must manually map Platform segments to the custom field attribute in [!DNL Zendesk].
+
+To do this, select each segment, then enter the corresponding custom field attribute from [!DNL Zendesk] in the **[!UICONTROL Mapping ID]** field.
+
+An example is shown below:
+![Platform UI screenshot example showing Schedule segment export.](../../assets/catalog/crm/zendesk/schedule-segment-export.png)
+
+## Validate data export {#exported-data}
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** and navigate to the list of destinations.
+1. Next, select the destination and switch to the **[!UICONTROL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/crm/zendesk/destinations-activation-data.png)
```

Comment:
Let's improve screenshots by eliminating anything that takes the attention away from the AEP UI. Please crop this screenshot so it doesn't show other browser toolbars and such.

Reply:
Apologies, thank you for noticing.